src/config/constants/index.ts

```diff
@@ -8,16 +8,21 @@ export const TOKEN_ADDRESSES = {
   MooCoin: '0xe62e4B079D40CF643D3b4963e4B675eC101928df',
   SeeroCoin: '0x3730954eC1b5c59246C1fA6a20dD6dE6Ef23aEa6',
   KibaCoin: '0x3D4AA21e8915F3b5409BDb20f76457FCdAF8f757',
-  ETH: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
-  WETH: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
-  DAI: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
-  USDC: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
-  SUSD: '0x57ab1ec28d129707052df4df418d58a2d46d5f51',
-  LUSD: '0x5f98805a4e8be255a32880fdec7f6728c6568ba0',
-  ALUSD: '0xbc6da0fe9ad5f3b0d58160288917aa56653660e9',
-  RAI: '0x03ab458634910aad20ef5f1c8ee96f1d6ac54919',
-  USDT: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
-  WBTC: '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599',
+  ETH: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'.toLowerCase(),
```

Comment:
@kibagateaux This felt a bit janky to me, but I was not sure what the better option was. The `tokensMap` from the Yearn SDK used to pull token data used a case-sensitive `address` key.